### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/supervisor/commit/c1244c021795cdb301b18136b045826023d72e6d#commitcomment-80692389

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/supervisor/commit/80e70d01ddb03627a6cdf66164d8dfce8a777d35

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1725,7 +1725,7 @@
 
 - Processes which started successfully after failing to start
   initially are no longer reported in BACKOFF state once they are
-  started successfully (thanks to Damjan from Macdonia for the bug
+  started successfully (thanks to Damjan from Macedonia for the bug
```
Comment (jsoref) [inline CHANGES.rst:1728]: I traced this back a ways, there are two parallel lines of text and the other spells it this way...

File: supervisor/rpcinterface.py [hunk 2]
```diff
@@ -60,7 +60,7 @@ def _update(self, text):
     def getAPIVersion(self):
         """ Return the version of the RPC API used by supervisord
 
-        @return string version version id
+        @return string version id
```
Comment (jsoref) [inline supervisor/rpcinterface.py:63]: sometimes doubled words are intentional. I thought about this one a bit and don't think that this doubling adds anything and it doesn't seem to match other forms in this file...

**Review by jsoref — COMMENTED** [review]
Most fixes suggested by Google Sheets. All fault mine.

**Comment by mnaberez** [comment]
> Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

I'm not interested in adding this to the workflow at this time.

**Pull request merged**